libvast/include/vast/tag.hpp

```diff
@@ -0,0 +1,35 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <variant>
+
+namespace vast {
```

Comment:
Style nit: Empty line around namespaces, i.e., one after this and one below the matching brace at the end.